Clear up dotnet tool sdk requirements (#629)
OctopusDeploy/docs: docs/octopus-rest-api/octopus-cli/install-global-tool.md

position: 0
---

> Microsoft introduced the ability to install tools using the .NET Core CLI called [Global Tools](https://docs.microsoft.com/en-us/dotnet/core/tools/global-tools). This lets you install the Octopus CLI onto machine or build agent as long as you have the .NET Core SDK available.

Reviewer: While we're here could we simplify this?
```suggestion
You can install the Octopus CLI as a .NET Core [Global Tool](https://docs.microsoft.com/en-us/dotnet/core/tools/global-tools). This requires that you have the [.NET Core Runtime](https://dotnet.microsoft.com/download/dotnet-core/current/runtime) installed.
```
